README.md

```diff
@@ -18,6 +18,13 @@ A detailed description can be found here: [Design Article](performance_test/desi
 
 ROS 2: https://github.com/ros2/ros2/wiki/Installation
 
+Additional dependencies are Boost and Java
+```
```

Comment:
Since boost is declared as a dependency in `package.xml`, this can be made more future-proof (e.g. if we add another dependency later on):

Install additional dependencies:

```
rosdep install -y --from performance_test --ignore-src
sudo apt-get install default-jre
```